src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreationParameters.java

```diff
@@ -11,29 +11,22 @@
  */
 public class AcEquationSystemCreationParameters {
 
-    private final boolean voltageRemoteControl;
-
     private final boolean phaseControl;
 
     private final boolean transformerVoltageControl;
 
     private final boolean forceA1Var;
 
-    public AcEquationSystemCreationParameters(boolean voltageRemoteControl, boolean phaseControl, boolean transformerVoltageControl) {
-        this(voltageRemoteControl, phaseControl, transformerVoltageControl, false);
+    public AcEquationSystemCreationParameters(boolean phaseControl, boolean transformerVoltageControl) {
```

Comment:
Ok so remote voltage control de-activation is only managed at network creation.

Reply:
Indeed, after that we do not need that boolean anymore.